Add a cursor for the current page. (#43)
mongoid/mongoid-scroll: README.md

end
```

> `saved_iterator.current_cursor` can be used to loop over the same records again.

Reviewer: ```suggestion
Use `saved_iterator.current_cursor` to loop over the same records again.
```

## Cursors

You can use `Mongoid::Scroll::Cursor.from_record` to generate a cursor. A cursor points at the last record of the 

> 

Reviewer: Let's get rid of the extra space ;) generally don't wrap markdown either 
https://code.dblock.org/2021/06/07/to-wrap-or-not-to-wrap-in-markdown.html
